includes/admin/feedzy-rss-feeds-import.php

```diff
@@ -2019,7 +2038,10 @@ private function try_save_featured_image( $img_source_url, $post_id, $post_title
 				return false;
 			}
 
-			$type = mime_content_type( $local_file );
+			$type = $this->get_file_type_by_url( $img_source_url );
```

Comment:
Everything looks good. 

I have a small optimization suggestion if you see the value in it. 

My suggestion would be first check if the `mime_content_type` function exists and if it does use it to get the mime type. If after this step the mime type is still empty then use the `get_file_type_by_url` function as this function will use more resources and we can avoid it if `mime_content_type` is present.

Reply:
@GrigoreMihai thank you for the suggestion. I've made the changes, let me know if this looks good to you now.

Reply:
@preda-bogdan just a small thing, from what I know the check here will generate a notice https://github.com/Codeinwp/feedzy-rss-feeds/pull/912/commits/b02f7a6753996fd15068f3957ee3c3a106055c82#diff-ac43a0e8339b0e90d3b294b426f835842f98839ea6679134e46ebe11b4faaeabR2047 if the variable is undefined which could happen.

I suggest to initialize `$type` with an empty string, unless there is something that I'm missing.